master-libvirt/master.cfg

```diff
@@ -120,66 +120,6 @@ def getRpmInstallStep():
     )
 
 
-def getDebGaleraStep(port):
-    def if_run_galera_test(step):
-        if step.getProperty("sst_mode") == "off":
-            return False
-        return True
```

Comment:
are you sure this can be removed entirely? It's been a while since I looked on libvirt, but for certain everything after this `return True` until line 143 since it's useless. Or we no longer run the `./deb-galera.sh` script?

Reply:
ok